components/contribution-flow/StepDetails.js

```diff
@@ -35,6 +37,13 @@ const FrequenciesI18n = defineMessages({
   },
 });
 
+const MinAmount = styled(Span)`
+  &:before {
+    content: 'Minimum Amount: ';
+    margin-left: -90px;
+  }
```

Comment:
It's a bit hacky to do it like that, and we need the strings to be internationalized (see [/help/developers/translations](https://docs.opencollective.com/help/developers/translations)).

Let's rather put the message in the DOM, and use flex/margins for positioning.

Reply:
@Betree Updated :+1: 

![Screenshot at 2020-06-04 16-03-10](https://user-images.githubusercontent.com/46647141/83748137-5b3cf380-a67f-11ea-95be-0e9b7a30d8df.png)